src/components/Card/Card.tsx

```diff
@@ -24,6 +24,7 @@ export const Card: FC<CardProps> = React.forwardRef(
             bodyListOneProps,
             bodyListTwoProps,
             children,
+            classNames,
```

Comment:
Did you also test the other shared props? styles and 'data-test-id'? You might consider extending from OcBaseProps and using a `...rest` spread.